### DISCUSSION TITLE
Adding user_id to Token Tracing with litellm and langfuse as a proxy

### DISCUSSION THREAD

**Discussion by octadion** [BerriAI/litellm#5041] [Q&A] [answered]
I'm currently using litellm as a proxy with langfuse for tracing tokens in MS GraphRAG. I want to include a user_id in the tracing process to better track and manage user interactions.

Could anyone guide me on how to add or configure the user_id within this setup? Any advice or examples would be greatly appreciated!

Thanks in advance

**Comment by ishaan-jaff** [comment, marked as answer]
pass `user` in all your /chat/completions request. LiteLLM will send this Langfuse

**Comment by cln-io** [comment]
I've recently upgraded LiteLLM from v1.52.16 to v1.57.4 and the `user` parameter stopped working for me.
(and langfuse from v2.93.4  to v2.93.8)
Troubleshooting it right now, will report back
